frontend/src/components/organisms/ViewEvents.tsx

```diff
@@ -396,22 +405,74 @@ const PastEvents = () => {
   /** Error screen */
   // if (isError) return <Error />;
 
+  // TODO: Replace constants with actual values
+  const hours = 36;
+  const REFERENCE_HOURS = 80;
+  const CERTIFICATE_HOURS = 120;
+
   return (
     <>
-      <Table
-        columns={volunteerEventColumns}
-        rows={pastVolunteerEvents.result}
-        dataSetLength={pastVolunteerEvents.total}
-        paginationModel={paginationModelVolunteer}
-        setPaginationModel={setPaginationModelVolunteer}
-      />
-      <Table
-        columns={SupervisoreventColumns}
-        rows={pastSupervisorEvents.result}
-        dataSetLength={pastSupervisorEvents.total}
-        paginationModel={paginationModelSupervisor}
-        setPaginationModel={setPaginationModelSupervisor}
-      />
+      {((role === "Admin" && pastSupervisorEvents.result.length == 0) ||
+        (role === "Supervisor" && pastSupervisorEvents.result.length == 0) ||
+        (role === "Volunteer" && pastVolunteerEvents.result.length == 0)) && (
+        <div className="p-10">
+          <div className="text-center">There are no past events</div>
+        </div>
```

Comment:
This looks better if it comes after the "reference and certificate hour tracker". Also, it looks like the table is showing anyway if the date is empty, but it is just a white strip that I think isn't that visually appealing

Reply:
<img width="1172" alt="Screenshot 2024-04-06 at 20 53 00" src="https://github.com/cornellh4i/lagos-volunteers/assets/59776300/099453b7-a82e-42c8-81e8-1f5bc43b228e">

Reply:
Good catch, fixed it.

![image](https://github.com/cornellh4i/lagos-volunteers/assets/48730262/8ea611ef-992d-47ec-aa93-10e68c0548b8)